index.html

```diff
@@ -227,7 +248,17 @@
                 <a id="dc-link" href="">Open chat</a>
             </li>
         </ol>
-        </section>
+
+        <details>
+            <summary>I have Delta Chat installed on another device</summary>
```

Comment:
maybe use instead:
```suggestion
            <summary>Click here if you have Delta Chat installed on another device</summary>
```

at least to me it looks a bit not obvious the text is clickable otherwise

Reply:
i added a commit, however, avoided to make the text longer.

moreover, i make the link look like a link on hover - i also tried some blueish, but that removes attention from the buttons, so, let's see.

it is still not clear if ppl are more consuming the link on the same or on another device, we'll know better if we have some feedback, maybe we can put the focus more on the QR code in general then